pyxrf/model/lineplot.py

```diff
@@ -363,6 +363,11 @@ def plot_experiment(self):
         """
         PLot raw experiment data for fitting.
         """
+
+        # Do nothing if no data is loaded
+        if self.data is None:
+            return
```

Comment:
Should this raise an error that gets caught further up and gives some feedback (like a dialog) to the user? Silently returning seems like it could lead to confusing situations.

Reply:
OK, that sounds sensible. Thanks for the response. :-D